examples/ge_example/setup.py

```diff
@@ -10,7 +10,7 @@
         "dagster",
         "dagit",
         "dagster-ge",
-        "great_expectations",
+        "great_expectations>=0.14.12",  # pinned because pip is using the cached wheel for 0.13.14
```

Comment:
i _ thin _ this is fine

Reply:
I really wasn't sure how pip was resolving it.   The logs show it downloading basically every wheel (including the latest `0.15.4`, and then it outputs:

```
[2022-05-10T01:28:27Z] ^[[?25h  Using cached great_expectations-0.13.14-py3-none-any.whl (4.7 MB)^M
```